src/sass/partials/_how-its-made.scss

```diff
@@ -0,0 +1,193 @@
+
+.section {
+    padding-top: 125px;
+    padding-bottom: 145px;
+    background-color: $VBgColor;
+    line-height: 1.2;
+    margin: 0 auto;
+}
+
+.how-its-made {
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+
+    &__title {
+        color: $IVColor;
+        font-family: $fontSecondary; 
+        font-size: 18px;
+        margin-bottom: 15px;
+        text-align: center;            
+        
+        &--ubig {
+            color: $IIIColor;
+            font-size: 40px;
+            margin-bottom: 26px;
+        }
+    }
+
+
+    &__section {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+
+
+    &__cow {
+        max-height:100%;
+        margin-bottom: 46px;
+    }
+
+    &__text-list {
+        display: flex;
+        flex-direction: column;
+        color: $IIColor;
+        font-family: $fontPrimary;
+        font-size: 12px;
+        font-weight: 500;
+        line-height: 1.6;
+        padding: 0 30px;
+        gap: 30px;
+        margin-bottom: 40px;
+        position: relative;
+        z-index: 99;
+
+        &--grey {
+            color: $textColor;
+        }
+    }
+
+    &__text-item {
+        display: flex;
+        flex-direction: column;
+        gap: 20px;
+    }
+
+    &__button {
```

Comment:
On the mobile version the "read me" button is 14px, if I check correctly.

Reply:
Heh, good eye :>